code/modules/heavy_vehicle/premade/powerloader.dm

```diff
@@ -173,3 +173,21 @@
 	..()
 	software = new(src)
 	software.installed_software = list(MECH_SOFTWARE_UTILITY, MECH_SOFTWARE_WEAPONS)
+
+/mob/living/heavy_vehicle/premade/ripley/remote
+	name = "remote power loader"
+	dummy_colour = "#ffc44f"
+	remote_network = "remotemechs"
+
+/mob/living/heavy_vehicle/premade/ripley/remote/Initialize()
+	. = ..()
+	become_remote()
```

Comment:
Might be worth to put that in the default initialize chain for the mechs with something like that:
```
if(remote_network)
    become_remote()
```